Simcoe/Adobe/Adobe.swift

```diff
@@ -38,29 +38,43 @@ extension Adobe: EventTracking {
     public func track(event: String,
                     withAdditionalProperties properties: Properties?) -> TrackingResult {
         ADBMobile.trackAction(event, data: properties)
+
         return .success
     }
 
 }
 
-// MARK: - LifetimeValueIncreasing
+// MARK: - LifetimeValueTracking
 
-extension Adobe: LifetimeValueIncreasing {
+extension Adobe: LifetimeValueTracking {
 
-    /// Increases the lifetime value of the key by the specified amount.
+    /// Tracks the lifetime value.
     ///
     /// - Parameters:
-    ///   - amount: The amount to increase that lifetime value for.
-    ///   - item: The optional item to extend.
+    ///   - key: The lifetime value's identifier.
+    ///   - value: The lifetime value.
     ///   - properties: The optional additional properties.
     /// - Returns: A tracking result.
-    public func increaseLifetimeValue(byAmount amount: Double, forItem item: String?, withAdditionalProperties properties: Properties?) -> TrackingResult {
-        var data = properties ?? Properties()
-        if let item = item {
-            data[item] = ""
+    public func trackLifetimeValue(_ key: String, value: Any, withAdditionalProperties properties: Properties?) -> TrackingResult {
+        var properties = properties ?? Properties()
+        properties[key] = ""
+
+        ADBMobile.trackLifetimeValueIncrease(NSDecimalNumber(value: value as? Double ?? 0), data: properties)
```

Comment:
May need context, but is 0 an appropriate default value or should this return as a failure if the value can't be casted as a Double?